mace/python/tools/converter_tool/base_converter.py

```diff
@@ -510,6 +517,11 @@ def build(self):
                 # Need to be put after SORT_BY_EXECUTION
                 TransformerRule.ADD_QUANTIZE_TENSOR_RANGE,
             ]
+            if self._device == DeviceType.APU.value:
```

Comment:
Place this transform in apu_converter.py?

Reply:
fixed in commit 689b236